spec/system/consumer/split_checkout_spec.rb

```diff
@@ -162,12 +162,23 @@
     context "on the 'details' step" do
       before do
         visit checkout_step_path(:details)
+        click_on "Checkout as guest"
       end
 
       it "should allow visit '/checkout/details'" do
         expect(page).to have_current_path("/checkout/details")
       end
 
+      it 'does not show the save as default bill address checkbox' do
```

Comment:
I think we should also test as connected user if the checkbox is well shown, right?

Reply:
I agree :+1: 

I did not add this test here explicitly as this other PR deals with the functionality of the [default bill address checkbox](https://github.com/openfoodfoundation/openfoodnetwork/pull/8960/files#diff-d09306155331299caecd70fa8c3f245c3e4c8d3b00f9a881015fafb3eb38cda8R217), as a connected user.

What do you think?

Reply:
Great idea!